modules/administration/pages/space-management.adoc

```diff
@@ -0,0 +1,68 @@
+[[space-management]]
+= Managing Disk Space
+
+If there is no available disk space, {productname} will not function.
```

Comment:
I think we must define what "available" means here.  "Enough disk space where {productname} can manage (write, store, delete, ...) data."  For this kind of oparation I used to use the term "free disk space".  Maybe, that's different now with all the new FSs such as btrfs.

Reply:
reworded.